### DISCUSSION TITLE
Swagger UI on Mockoon cli.

### DISCUSSION THREAD

**Discussion by shokri-navid** [mockoon/mockoon#1200] [Help]
I am using Mockoon to create and develop my product using the API first approach. I defined my exposed Apis and imported it to my Mockoon that is run on docker. I want to share this mocked API with other developers. So, I am looking for a swagger UI page. 
is there any ability to serve a swagger page on Mockoon or not?
if the answer is not please let me know the instruction and if it is No, is there any plan to add this feature?

**Comment by 255kb** [comment]
Mockoon has a history of not being an API design tool (i.e. it does not cover everything from the OpenAPI/Swagger specification, and will probably never do), but an API mocking first. This has consequences, like the lack of a request description. Because we don't really care about the request. There is currently no plan to go down this road and do everything, but I understand that it can be appealing! 
As a workaround, you can [export your Mockoon mock to an OpenAPI specification](https://mockoon.com/docs/latest/openapi/import-export-openapi-format/#export-an-environment-to-an-openapi-v3-json-file) (with some [limitations](https://mockoon.com/docs/latest/openapi/openapi-specification-compatibility/)) that you can read using [Swagger UI](https://swagger.io/tools/swagger-ui/).

**Comment by shokri-navid** [comment]
So, We found a way to create a swagger for our mocked API. currently, we are running the Mockoon on docker and we added a [swagger-UI ](https://hub.docker.com/r/swaggerapi/swagger-ui)container beside the Mockoon. and export a Swagger JSON from our Mockoon UI and then configure our Swagger-UI to read the given JSON file. so, finally, we have the swagger-UI
